tests/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 if (BUILD_TESTING)
-    add_subdirectory(../tests/testing-resources)
-    add_subdirectory(../tests/aws-cpp-sdk-core-tests)
+    add_subdirectory(testing-resources)
+    add_subdirectory(aws-cpp-sdk-core-tests)
```

Comment:
what about the rest?
I would suggest checking if bla-bla-bla-tests exists then add (to cover cases of missing or possible future or manually removed tests (or git sparse checkout).